test/compress/properties.js

```diff
@@ -31,7 +31,7 @@ dot_properties: {
         a.foo = "bar";
         a["if"] = "if";
         a["*"] = "asterisk";
-        a["\u0EB3"] = "unicode";
+        a.\u0EB3 = "unicode";
```

Comment:
This is actually legal in ES5, since `\u0EB3` is in the "Other letter (Lo)" Unicode category. Look for `UnicodeLetter` in [section 7.6](https://www.ecma-international.org/ecma-262/5.1/#sec-7.6)